src/jvm/test/org/jetbrains/kotlinx/lincheck_test/AbstractLincheckTest.kt

```diff
@@ -67,4 +67,4 @@ abstract class AbstractLincheckTest(
     }
 }
 
-private const val TIMEOUT = 10 * 60_000L // 10 min
\ No newline at end of file
+private const val TIMEOUT = 20 * 60_000L // 10 min
```

Comment:
please revert

Reply:
Otherwise, it fails because of time out on one of the tests

Reply:
Here [it](https://teamcity.jetbrains.com/buildConfiguration/KotlinTools_KotlinxLincheck_BuildLinuxOnJava21TraceDebugger/4969210?buildTab=tests&focusLine=NaN&name=SerializableJavaUtilParameterTest.testWithMod&expandedTest=build:(id:4969210),id:2000000297) is.